docs/contributor/storage/configuration.md

```diff
@@ -62,7 +69,37 @@ device-classes:
 ```
 Caveats:
 - Specifying lvcreate-options is at your own risk and only provided as an escape hatch by the CSI plugin. 
-- Setting spare-gb to anything other than 0 is not recommended because it does not behave predictably and usually results in more space being allocated that expected.
+- Setting spare-gb to anything other than 0 is not recommended because it does not behave predictably and usually results
+in more space being allocated that expected.
+
+#### LVM Thin Volumes
+
+Advanced storage features such as volume cloning and snapshotting are only supported on thin volumes, and thus require an
+LVM thin-pool on the host and the appropriate LVMS and cluster configuration.  For information on creating a thin-pool,
+see [OpenShift documentation](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_and_managing_logical_volumes/creating-and-managing-thin-provisioned-volumes_configuring-and-managing-logical-volumes).
+
+> VMs created via the [Getting Started](../../user/getting_started.md) instructions are provisioned with a single
+thin-pool automatically. 
+
+For LVMS to manage thin LVs, a thin-pool device class must be specified in the lvmd.yaml. Multiple thin-pool device classes 
+are permitted. LVM thin-pools must be attached to a volume group. However, it is not necessary to expose the volume-group 
+to LVMS unless thick volume provisioning is required as well.  For example, the following lvmd.yaml specifies a single 
+device class for a thin-pool:
```

Comment:
I don't understand what this means. Does it mean that only snapshots work? What about the example config means that the volume group not exposed to LVMS?

Reply:
"Expose" is poor word choice I think.  The idea was to simply point out that, for LVMS, thick LV provisioning is not a dependency for thin LV provisioning. It seemed worth pointing out at the time, but less so in retrospect.  I'll rework the paragraph to focus on thin-lv configs.